### DISCUSSION TITLE
Actividad 0 - Lopez Velazco Bruno Tobias

### DISCUSSION THREAD

**Discussion by brunolopezv** [frlp-utn-ingsoft/catedra#40] [Show and tell]
https://github.com/brunolopezv/actividad0

**Comment by ivanduva** [comment]
Buenas!
Está rota la lista de encuestas cuando quiero ingresar a `/polls`. Me tira un error en el template `index.html`.
Arreglalo para aprobar la actividad.
Lo demás está bien, la atomicidad de los commits y los mensajes.
Saludos!

**Reply by brunolopezv** [reply]
Hola Ivan!
Disculpame por el error, ya lo corregí.
Gracias por tus comentarios.
Saludos!

**Comment by brunolopezv** [comment]
Corregido,
Gracias!

________________________________

Bruno Lopez Velazco

________________________________
De: ivanduva ***@***.***>
Enviado: sábado, 13 de abril de 2024 17:44
Para: frlp-utn-ingsoft/catedra ***@***.***>
Cc: brunolopezv ***@***.***>; Author ***@***.***>
Asunto: Re: [frlp-utn-ingsoft/catedra] Actividad 0 - Lopez Velazco Bruno Tobias (Discussion #40)


Buenas!
Está rota la lista de encuestas cuando quiero ingresar a /polls. Me tira un error en el template index.html.
Arreglalo para aprobar la actividad.
Lo demás está bien, la atomicidad de los commits y los mensajes.
Saludos!

**Comment by RodrigoJacznik** [comment]
Ya esta fixiado.
La actividad esta aprobada.
Saludos
